pkg/helm/helm.go

```diff
@@ -50,6 +50,9 @@ var (
 
 	// IsHelmDebug is configured via a flag and enables extensive debug logging for Helm actions.
 	IsHelmDebug bool
+
+	//
+	installUpgradeTimeout = 150 * time.Second
```

Comment:
We do not have any metrics for the plugin upgrade duration. Let's stick with the 5m default used by Helm.